makefile

```diff
@@ -0,0 +1,3 @@
+tests:
+	go test -race -timeout 30s -cover ./cmd ./check
```

Comment:
This line fails. You don't have cmd and check packages. And the second go test command caters for all packages.